Adamant/Modules/Wallets/Ethereum/EthApiService.swift

```diff
@@ -12,6 +12,8 @@ import web3swift
 import Web3Core
 
 class EthApiService: ApiServiceProtocol {
+    static let currencySymbol = "ETH"
```

Comment:
Why is it in the ApiService? Looks like an error, it has to be in the EthWalletService